R/format_p.R

```diff
@@ -26,7 +26,7 @@ format_p <- function(p, stars = FALSE, stars_only = FALSE, name = "p", missing =
       ifelse(p < 0.001, "< .001***",
         ifelse(p < 0.01, "< .01**",
           ifelse(p < 0.05, "< .05*",
-            paste0("= ", insight::format_value(p, digits))
+            paste0("= ", insight::format_value(p, 3))
```

Comment:
Why this change?

Reply:
ok, I assume this is an APA requirement?

Reply:
Otherwise it fails because digits is a character ("apa") and format_value expects an int